Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -163,6 +183,7 @@ extension SwiftModule {
         buildSettings["PRODUCT_MODULE_NAME"] = c99name
         buildSettings["OTHER_SWIFT_FLAGS"] = "-DXcode"
         buildSettings["MACOSX_DEPLOYMENT_TARGET"] = "'10.10'"
+        buildSettings["LIBRARY_SEARCH_PATHS"] = "/usr/local/lib"
```

Comment:
I thought system modules will probably be present there on darwin, this would be handled better after the "SwiftPM System Module Search Paths" is accepted/implemented. Should I remove this for now?